docker-compose.yml

```diff
@@ -1,27 +1,22 @@
 version: "3"
 services:
   localstack:
-    image: localstack/localstack
+    image: localstack/localstack:0.10.5
```

Comment:
I really couldn't get 0.11.2 to work locally so have shifted to version that works for me